app/components/edit_channel_info/index.js

```diff
@@ -130,7 +130,7 @@ export default class EditChannelInfo extends PureComponent {
             return;
         }
 
-        const displayNameExists = displayName && displayName.length >= 2;
+        const displayNameExists = displayName && displayName.length >= 1;
```

Comment:
Do you mind using here also the constant?

Reply:
Not at all, fixed in latest commit.